api/yangSearch/mysql_index.py

```diff
@@ -133,4 +133,3 @@ def do_search(options, dbHost, dbName, dbPass,  dbUser):
         conn.close()
         raise Exception("Error searching for {}: {}".format(
             opts['search'], e.args[0]))
-
```

Comment:
do not make format changes on files that you do not change at all.

on the other hand. This file can be removed now completely